src/components/Button/Button.tsx

```diff
@@ -1,37 +1,74 @@
+import "./Button.scss";
 import { ReactNode } from "react";
 import cx from "classnames";
-import "./Button.scss";
+import ButtonLink from "./ButtonLink";
+
+type ButtonVariant = "primary" | "primary-action" | "semi-clear" | "clear";
 
-type Props = {
+type ButtonProps = {
   children: ReactNode;
-  imgSrc?: string;
-  imgName?: string;
+  variant: ButtonVariant;
   className?: string;
-  href?: string;
-  onClick?: () => void;
   disabled?: boolean;
+  onClick?: () => void;
+  to?: string;
+  type?: "button" | "submit" | "reset";
+  imgInfo?: {
+    src: string;
+    alt?: string;
+  };
 };
 
-export default function Button({ href, imgSrc, children, className, imgName, onClick, disabled = false }: Props) {
-  let classNames = cx("btn", className);
+export function getClassName(variant: ButtonVariant) {
```

Comment:
mm, do we need to export this function)?

Reply:
Removed!